internal/pkg/project/project.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package project
```

Comment:
Why do we need this package?  It seems like a shallow wrapper around the `store` package

Reply:
I wanted a single dependency than can talk projects without needing to initialize an `environmentGetter` **and** a `projectGetter` **and** an `applicationGetter`. I think this will be reusable across commands where we need to source input that relates to existing workflows.

Reply:
Could we not create an interface instead in the `cli` package for all these getters?

```go
type storeGetter interface {
  archer.ProjectGetter
  archer.EnvironmentGetter
  archer.ApplicationGetter
}
```

Reply:
I thought we wanted to move away from the `archer` package in favor of defining the abstractions where we use them in the `cli` package. I can consume the `archer` package if that's not the case.

Reply:
Yeah unfortunately extracting those interfaces and structs outside to the cli package might be too much work, I think it's fine if we take a dependency to the `archer` pkg for now